rlkit/envs/primitives_make_env.py

```diff
@@ -90,6 +91,10 @@ def make_base_metaworld_env(env_name, env_kwargs=None, use_dm_backend=True):
     else:
         env = env_cls(seed=42)
     env.reset_action_space(**action_space_kwargs)
+
+    if isinstance(env, SawyerMocapBaseDMBackendMetaworld):
+        env.use_wrist_cam = env_kwargs['use_wrist_cam']
```

Comment:
this is unnecessary pass this arg through `action_space_kwargs`

Reply:
Fixed, I've added it to action_space_kwargs